### PR TITLE
Add ChakraUI&Eslint+Typescript fixes

### PR DESCRIPTION
**Pull request by leonardomolinan**
:hammer_and_wrench: **Changes:**
- [@typescript-eslint/parser](https://www.npmjs.com/package/@typescript-eslint/parser) added
- fix on _.eslintrc.json_ `prettier/@typescript-eslint`, that was merged on `prettier` ([you can see here](https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21)) 
- `@chakra-ui` basic config using `@emotion/react@^11` and `@emotion/styled@^11` following the [Getting Started](https://chakra-ui.com/docs/getting-started)

### CODE REVIEW

**Comment by diogomqbm** [comment]
I think the theme file is missing, we should do an initial configuration for this.

**Review by diogomqbm — APPROVED** [review]

File: template/package.json [hunk 1]
```diff
@@ -3,13 +3,18 @@
...
+    "@emotion/styled": "^11",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "@types/jest": "^26.0.15",
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "@typescript-eslint/parser": "^4.19.0",
+    "framer-motion": "^4",
```
Comment (diogomqbm) [inline template/package.json:17]: Why did you add `framer-motion`?

Comment (leonardomolinan) [reply template/package.json:17]: It's on the [Getting Started](https://chakra-ui.com/docs/getting-started) from `@chakra-ui`
